Tabloid/client/src/components/AddReactions.js

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import "../index.css";
+import { addReaction } from "../modules/reactionManager";
+import { useHistory } from "react-router-dom";
+
+export const AddReactions = () => {
+    const [reaction, setReaction] = useState({ name: null, imageLocation: null })
+    const reactionUrl = "/api/reaction";
+    const history = useHistory()
+
+    const handlePost = (e) => {
+        e.preventDefault();
+        if (reaction.name == null || reaction.imageLocation == null) {
+            window.alert("Please fill out all input fields");
```

Comment:
Could we add validation instead? maybe disable the submit button if all fields aren't filled out?

Reply:
So I feel like this might have gotten worse, not better. We do want the user to know that the button doesn't do anything, so minimally a visual cue that the button isn't going to work is important (by using the `disabled` attribute for example).